slither-reports/reports.txt

```diff
@@ -0,0 +1,118 @@
+Low level call in Address.functionStaticCall(address,bytes,string) (node_modules/@openzeppelin/contracts/utils/Address.sol#157-166):
+        - (success,returndata) = target.staticcall(data) (node_modules/@openzeppelin/contracts/utils/Address.sol#164)
+Low level call in Address.functionDelegateCall(address,bytes,string) (node_modules/@openzeppelin/contracts/utils/Address.sol#184-193):
+        - (success,returndata) = target.delegatecall(data) (node_modules/@openzeppelin/contracts/utils/Address.sol#191)
+Low level call in EthernalGates.withdraw() (contracts/EthernalGates.sol#76-79):
+        - (success) = address(msg.sender).call{value: address(this).balance}() (contracts/EthernalGates.sol#77)
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#low-level-calls
+
+Parameter EthernalGates.setSaleStatus(EthernalGates.SALE_STATUS)._status (contracts/EthernalGates.sol#54) is not in mixedCase
+Parameter EthernalGates.setMerkleRoot(bytes32)._merkleRoot (contracts/EthernalGates.sol#58) is not in mixedCase
+Parameter EthernalGates.changePresalePrice(uint256)._presalePrice (contracts/EthernalGates.sol#62) is not in mixedCase
+Parameter EthernalGates.changePublicPrice(uint256)._publicPrice (contracts/EthernalGates.sol#66) is not in mixedCase
+Parameter EthernalGates.increaseSupply(uint256)._increaseBy (contracts/EthernalGates.sol#71) is not in mixedCase
+Parameter EthernalGates.reserveEthernalGates(uint256)._count (contracts/EthernalGates.sol#82) is not in mixedCase
+Parameter EthernalGates.airdrop(address[],uint256)._addresses (contracts/EthernalGates.sol#86) is not in mixedCase
+Parameter EthernalGates.airdrop(address[],uint256)._count (contracts/EthernalGates.sol#86) is not in mixedCase
+Parameter EthernalGates.presaleMint(bytes32[],uint256,uint256)._proof (contracts/EthernalGates.sol#119) is not in mixedCase
+Parameter EthernalGates.presaleMint(bytes32[],uint256,uint256)._allowedCount (contracts/EthernalGates.sol#120) is not in mixedCase
+Parameter EthernalGates.presaleMint(bytes32[],uint256,uint256)._count (contracts/EthernalGates.sol#121) is not in mixedCase
+Parameter EthernalGates.publicMint(uint256)._count (contracts/EthernalGates.sol#159) is not in mixedCase
+Parameter EthernalGates.mint(address,uint256)._addr (contracts/EthernalGates.sol#165) is not in mixedCase
+Enum EthernalGates.SALE_STATUS (contracts/EthernalGates.sol#11-17) is not in CapWords
+Parameter ERC721A.safeTransferFrom(address,address,uint256,bytes)._data (node_modules/erc721a/contracts/ERC721A.sol#313) is not in mixedCase
+Variable ERC721A._currentIndex (node_modules/erc721a/contracts/ERC721A.sol#67) is not in mixedCase
+Variable ERC721A._burnCounter (node_modules/erc721a/contracts/ERC721A.sol#70) is not in mixedCase
+Variable ERC721A._ownerships (node_modules/erc721a/contracts/ERC721A.sol#80) is not in mixedCase
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#conformance-to-solidity-naming-conventions
```

Comment:
These are all styles nits that we said in our report that we would mention here using slither.